data/database.json

```diff
@@ -35,7 +35,7 @@
     ],
     "letterGrades":
     [
-        {"id":432483, "letterGrade":"B"},
```

Comment:
Please do not delete a letter grade from the database.json like this. You can change the letter grade to the "Not Submitted Yet" string which indicates that a letter grade has not been submitted yet. Deleting this field from the json file will break the database